src/public/modules/core/components/toolset-dropdown.client.component.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+angular.module('core').component('toolsetDropdownComponent', {
+  templateUrl: 'modules/core/componentViews/toolset-dropdown.html',
+  controller: ['$scope', toolsetDropdownController],
+  controllerAs: 'vm',
+})
+
+function toolsetDropdownController($scope) {
+  /**
+   * Logic for dropdown component. Add to labelAndLink to add more websites to the dropdown.
+   */
+  const vm = this
+
+  vm.isDropdownHover = false
+  vm.isDropdownFocused = false
+  vm.isDropdownOpen = false
+  vm.labelAndLink = [
+    { label: 'Short Links', link: 'https://go.gov.sg/' },
+    { label: 'Mass Messaging', link: 'https://go.gov.sg/postmangovsg' },
+    { label: 'Eligibility Checker', link: 'https://go.gov.sg/checkfirstgovsg' },
+    { label: 'Voucher Distribution', link: 'https://go.gov.sg/redeemgovsg' },
+    { label: 'Website Builder', link: 'https://go.gov.sg/isomergovsg' },
+    { label: 'Payments', link: 'https://go.gov.sg/pay' },
+    { label: 'Citizen Identity', link: 'https://go.gov.sg/idgovsg' },
```

Comment:
Are all indirections through go.gov.sg for all the product links really needed? (e.g. `https://go.gov.sg/redeemgovsg` vs. `https://redeem.gov.sg/`)

Reply:
Hi, I just followed the specifications laid out by issue #2816 . Has there been a consensus made on whether to use the direct link or go.gov?

Reply:
let's just follow the spec, no major difference